src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+public class Ui {
+    private String horizontalLine = "____________________________________________________________\n";
```

Comment:
`horizontalLine` is a constant here hence by coding standards should be in all caps.
```suggestion
    private final String HORIZONTAL_LINE = "____________________________________________________________\n";
```
Similar comment for the `logo` string